README.md

```diff
@@ -655,23 +652,23 @@ In the below example, `result.context` would be `testing > rocks`.
 
 ```javascript
 o.spec("testing", function() {
-	o.spec("rocks", function() {
-		o(false).equals(true)
-	})
+    o.spec("rocks", function() {
+        o(false).equals(true)
+    })
 })
 ```
 
 ---
 
 ## Run time model
 
-### Definitions: 
+### Definitions
 
 - A **test** is the function passed to `o("description", function test() {})`.
 - A **hook** is a function passed to `o.before()`, `o.after()`. `o.beforeEach()` and `o.afterEach()`
 - A **task** designates either a test or a hook.
-- A given test and its associated `beforeEach` and `afterEach` hooks form a **streak**. The `beforeEach` hooks run outermost first, the `afterEach` run outermost last. The hooks are optional, and are tied at test-definition time in the `o.spec()` calls that enclose the test.
-- A **spec** is a collection of streaks, specs, one `before` *hook* and one `after` *hook*. Each component is optional. Specs are defined with the `o.spec("spec name", function specDef() {})` calls.
+- A **streak** is a given test and its associated `beforeEach` and `afterEach` hooks. The `beforeEach` hooks run outermost first, the `afterEach` run outermost last. The hooks are optional, and are tied at test-definition time in the `o.spec()` calls that enclose the test.
+- A **spec** is a collection of streaks, specs, one `before` hook and one `after` hook. Each component is optional. Specs are defined with the `o.spec("spec name", function specDef() {})` calls.
```

Comment:
While we're on the subject, let's try cleaning it up even further.

```suggestion
- A **streak** is a given test and its optional, associated `beforeEach` and `afterEach` hooks declared in an enclosing spec. The `beforeEach` hooks run outermost to innermost, then the test executes, then the `afterEach` run innermost to outermost.
- A **spec** is a collection of optional streaks, specs, `before` hooks, and `after` hooks. The `before` hooks are run, then the inner tests and specs execute in order, then the `after` hooks are run. Specs are defined via `o.spec("spec name", function specDef() {})`.
```

Reply:
_(got it; my goal was to leave verbiage + voice ~ close to original; your edits bring up opportunity for additional clarity; how about this:)_

### Definitions

- EDITED: A **test** is the function passed to `o("description", function() {})`.

- EDITED: A **hook** is the function passed to `o.before()`, `o.after()`, `o.beforeEach()` or `o.afterEach()`. Hooks are optional.

- EDITED: A **task** is either a test or a hook.

- ADDED: An **assertion** is a claim expressed as o(_value1_)._condition_.(_value2_). _condition_ may be `equals`, `notEquals`, `deepEquals` or `notDeepEquals`. Assertions may be used in any task, but usually occur in a test.

- EDITED: A **streak** is a test, and any `beforeEach` and `afterEach` hooks declared in enclosing scope(s)[ed. as a matter of definition-intent unsure whether should be "enclosing scope(s)" or "immediately-enclosing scope"; i suggest former]. A scope is a containing spec, or global-scope if no spec is present. During execution `beforeEach` hooks run outermost to innermost, then the test runs, then `afterEach` hooks run innermost to outermost.

- EDITED: A **spec** is a collection of streaks, specs, and any `before` and `after` hooks declared in enclosing scope(s)[ed. same q as above]. Specs are defined by calling `o.spec("spec name", function() {})`. During execution `before` hooks run outermost to innermost, then inner streaks and specs run in order of definition, then `after` hooks run innermost to outermost.

Note:

IMO "spec" is doing double-duty above, meaning both a passed-in function, and its runtime semantics; contrast w/ "test" (is a function), "streak" (is a test's runtime semantics). This double-duty is ok, and congruent w/ other overloaded entities in ospec, but how about this:

- EDITED: A **spec** is the function passed to `o.spec("spec name", function() {})`. It is a collection of [its own ?] hooks, streaks, specs, and any `before` and `after` hooks declared in enclosing scope(s)[ed. same q as above]. During execution `before` hooks run outermost to innermost, then inner streaks and specs run in order of definition, then `after` hooks run innermost to outermost.

Reply:
> IMO "spec" is doing double-duty above, meaning both a passed-in function, and its runtime semantics; contrast w/ "test" (is a function), "streak" (is a test's runtime semantics). This double-duty is ok, and congruent w/ other overloaded usages in ospec, but how about this:

I would be very open with revising the terminology, but I'd like to discuss that in a dedicated issue before accepting a pull request as it'd result in a massive overhaul across much of the repo's README.

> - EDITED: [...]

Very close on the last bit. What about this?

> A **spec** is the function passed to `o.spec("description", function() {})`. It is a collection of streaks, specs, hooks, and any `before` and `after` hooks declared directly within the function. During execution, `before` hooks run outermost to innermost, then inner streaks and specs run in order of definition, then `after` hooks run innermost to outermost.

Diff:

> A **spec** is the function passed to `o.spec("description", function() {})`. It is a collection of streaks, specs, <del>[its own ?] </del>hooks, and any `before` and `after` hooks declared <del>in enclosing scope(s)[ed. same q as above]</del><ins>directly within the function</ins>. During execution<ins>,</ins> `before` hooks run outermost to innermost, then inner streaks and specs run in order of definition, then `after` hooks run innermost to outermost.